packages/kit/src/views/Setting/pages/Tab/SubSettingsPage.tsx

```
@@ -0,0 +1,67 @@
+    <Page>
+      <Page.Header title={name} />
+      <Page.Body>
+        <ScrollView
+          keyboardShouldPersistTaps="handled"
+          contentContainerStyle={{ pb: '$10' }}
+        >
+          <YStack gap="$4" px="$4" pt={isTabNavigator ? undefined : '$3'}>
+            {configList?.map((item) => {
+              const list = Array.isArray(item) ? item.filter(Boolean) : [];
+              return list.length ? (
+                <TabSettingsSection>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)